### PR TITLE
[systemtest] Update KafkaST's testLabelsAndAnnotationForPVC

### PR DESCRIPTION
**Pull request by im-konge**
Signed-off-by: Lukas Kral <lkral@redhat.com>

### Type of change

- Enhancement

### Description

This PR will add Kubernetes labels and check to KafkaST's labels test.

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST#testLabelsAndAnnotationForPVC

File: systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java [hunk 1]
```diff
@@ -2064,6 +2073,12 @@ void testLabelsAndAnnotationForPVC() {
             .endSpec()
             .done();
 
+        LOGGER.info("Check if Kubernetes labels are applied");
+        Map<String,String> actualStatefulSetLabels = KafkaResource.kafkaClient().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get()
+                .getSpec().getKafka().getTemplate().getStatefulset().getMetadata().getLabels();
+        assertThat(actualStatefulSetLabels, is(statefulSetLabels));
```
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java:2079]: Does this work? I thought that the `app.kubernetes.io/managed-by` should not be configurable by user.

Comment (im-konge) [reply systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java:2079]: TBH I found that on Kubernetes website and tried that -> both labels were set

Comment (im-konge) [reply systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java:2079]: @scholzj is it some kind of issue or is it correct?

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST#testLabelsAndAnnotationForPVC

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST#testLabelsAndAnnotationForPVC

**Pull request merged**